src/widgets/PrescriptionSummary.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import { View, Text, StyleSheet, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { Seperator } from './Seperator';
+import { PrescriptionSummaryRow } from './PrescriptionSummaryRow';
+
+import { APP_FONT_FAMILY, SUSSOL_ORANGE } from '../globalStyles';
+
+/**
+ * Simple layout component, rendering a title and a list of summary
+ * rows on an elevated view.
+ *
+ * @param {Object} transaction
+ */
+export const PrescriptionSummary = ({ transaction }) => (
+  <View style={localStyles.containerStyle}>
+    <Text style={localStyles.titleStyle}>Item Details</Text>
```

Comment:
I've added this PR number to the #1693 issue.
`Item Details` could do with translation